src/components/TableSimple.stories.tsx

```diff
@@ -0,0 +1,143 @@
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+import { TableSimple } from './TableSimple';
+
+export default {
+  title: 'Components/TableSimple',
+  component: TableSimple
+} as ComponentMeta<typeof TableSimple>;
+
+const Template: ComponentStory<typeof TableSimple> = arguments_ => (
+  <TableSimple {...arguments_} />
+);
+
+// More on args: https://storybook.js.org/docs/react/writing-stories/args
+export const Standard = Template.bind({});
+Standard.args = {
+  caption: (
+    <>
+      This is the{' '}
+      <a href='https://cfpb.github.io/design-system/components/tables#standard-table-1'>
+        default style
+      </a>{' '}
+      at the large screen breakpoint. Tables are not responsive by default;
+    </>
+  ),
```

Comment:
Storybook's rendering of `ReactElement` args drives me crazy but I couldn't find a better alternative so 🤷 

<img width="938" alt="image" src="https://user-images.githubusercontent.com/1060248/229864172-0b0cb1ab-61c3-4c9d-ac19-f4b353f46720.png">